sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/ShareAsyncClient.java

```diff
@@ -152,6 +153,57 @@ public ShareFileAsyncClient getFileClient(String filePath) {
             serviceVersion);
     }
 
+    /**
+     * Gets if the share this client represents exists in the cloud.
```

Comment:
This would differ from the other Javadocs but I feel it reads better.

```suggestion
     * Determines if the share this client represents exists in the cloud.
```